app/assets/src/components/ui/containers/Modal.jsx

```diff
@@ -10,8 +10,11 @@ class Modal extends React.Component {
   }
 
   render() {
+    // The Semantic UI Modal component always expects a 'trigger', even though it always doesn't make sense for us.
```

Comment:
"always doesn't" => "never"
or did you meant "does not always"?

Reply:
I meant "doesn't always". Will fix.